display.go

```diff
@@ -229,7 +249,7 @@ func updateStaticContents() {
 	nativeInstance.UpdateLabelAndChangeVisibility("build_date", version.BuildDate)
 	nativeInstance.UpdateLabelAndChangeVisibility("golang_version", version.GoVersion)
 
-	// nativeInstance.UpdateLabelAndChangeVisibility("boot_screen_device_id", GetDeviceID())
+	nativeInstance.UpdateLabelAndChangeVisibility("device_id", GetDeviceID())
```

Comment:
Should we use `UISetVar` here too and remove `UpdateLabelAndChangeVisibility` completely?
EEZ has the ability to change visibility based on a variable, but I'm not sure whether we want to complicate the EEZ code or the Golang code.

Reply:
I think absolutely that would be much simpler! I just didn't know if you folks would mind that big a change. I'll do it (my) Friday night.

Reply:
Sure thing.

Any thoughts about Adam's comments? Do we need to reenable and of those other screens?